### DISCUSSION TITLE
Configuring the project

### DISCUSSION THREAD

**Discussion by Jellybeanz1980** [ehorvat1/NES-MEP-Reader#3] [Q&A] [answered]
Hello,

I'd like to try your project on my custom board, but I couldn't figure out how to configure Visual Studio Code (VSC) / PlatformIO. I'm using ESP32 (ESP32-WROOM-32E). I downloaded the repository, and opened the directory in VSC. The PlatformIO returns errors during loading, saying that some ESP32 related packages could not be downloaded. Could you provide a little more detail about loading and generating binary code for uploading to the board?

**Comment by ehorvat1** [comment, marked as answer]
Hi Jellybeanz1980,
I started to work with VSC last year when I started this project. So I am not the real expert for VSC/Platformio setup....
I am running VSC on a Linux system.
Maybe I was just lucky that VSC did work out of the box.
All I did was to install VSC , added the PlatformIO addon and used the entire folder (File - open Folder).

You must be connected to internet so VSC/PlatformIO is able to load correct packages.

Since I just change/update to new version please try V1.0 from the releases section . Or try to start with test compiling the "base" project from where I forked from for having a base to start from : https://github.com/UtilitechAS/amsreader-firmware .

I am using the file "platformio.ini" like it is in source code, I have not changed this at all.
So my routine to compile is : Open VSC, open Project Folder (File - open Folder - point to base Folder where platformio.ini is located) - when you do this 1st time it will take a minute or so updating VSC/Platformio stuff. Then you should get a "Project has been successfully updated!" message in the PlatformIO Output panel.
BR, Engelbert
